config/compute/config.go

```diff
@@ -446,6 +446,10 @@ func Configure(p *config.Provider) { //nolint: gocyclo
 		r.References["router"] = config.Reference{
 			Type: "Router",
 		}
+		r.References["subnetwork"] = config.Reference{
+			Type:      "Subnetwork",
+			Extractor: common.PathSelfLinkExtractor,
+		}
```

Comment:
Why did we add this configuration and where do we use it?

Reply:
I can remove this. During some initial testing I was having an issue with the RouterInterface resource. I can open another PR in the future if I hit the error again.